gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,63 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\]'
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Sender -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'HEIGHT: [0-9]+'
+stdout 'EVENTS: \[\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\}(?:,\{("type":"\w+","pkg_path":".*?","identifier":"\w+","attributes":\[(?:\{"key":"\w+","value":"\w+"\}(?:,)?)*\])\})*\]'
```

Comment:
Could you avoid using regex when it's not needed?

Reply:
Please compare stdout using `cmp stdout stdout.golden`, as many other tests do.

Reply:
Resolving this because @notJoon completely overhauled the initial txtars over the course of this PR to rely less on regex, and more on actual expected output

We will leave the golden comparisons to the txtar cleanup in a future PR 🙏